### ISSUE TITLE
Purchasing Cache 2022 did not give me battle pass points 

### ISSUE THREAD

**Issue by BHananaXD** [ValveSoftware/Dota2-Gameplay#5877]
### Description

I bought 1 of the cache 2022 chest opened it I didn't get the points (2500). I recycled the item for 2 battle pass levels yes i didn't receive yet the 2500 that was meant to be given when you open a chest.

### Example Match ID (and possibly Timestamp)

12-9-2022 8:20 PM MNL time

### Screenshots

![Pointns 2](https://user-images.githubusercontent.com/120148008/206602729-014b7492-3ca8-4db5-b2b3-dc4903a6df3c.PNG)
![no poinnts](https://user-images.githubusercontent.com/120148008/206602732-5d0bb6c8-9ff9-49ca-8a2d-c660108b5849.PNG)

**Comment by TheUnholyWrath** [comment]
You can only receive the rewards for opening cache once. The reward boxes are lit up as if you've already received them. If you haven't already opened 14 then theres a bug, if not please close this issue.

**Comment by BHananaXD** [comment]
oh i see so if i opened 14 before i will not get it anymore ? @TheUnholyWrath

**Comment by TheUnholyWrath** [comment]
> oh i see so if i opened 14 before i will not get it anymore ? @TheUnholyWrath

yeah it works differently to the immortal treasures, you can only get this reward once.
